spec/graphql/upgrader/member_spec.rb

```diff
@@ -213,10 +213,87 @@ class UserInterface < BaseInterface
        end
       }
       new = %{
-       field :name, String, field: SomeField, null: false do
+       field :name, String, field: SomeField, null: true do
        end
       }
       assert_equal upgrade(old), new
     end
   end
+
+  describe 'complicated inline definition' do
+    # it 'upgrades without breaking syntax' do
+    #   old = %{
+    #     connection :example_connection, -> { ExampleConnectionType } do
+    #       argument :order_by, (GraphQL::InputObjectType.define do
+    #         name 'ExampleConnectionOrder'
+    #         argument :direction, -> { !ExampleConnectionDirectionEnum }
+    #         argument :field, -> { !ExampleConnectionFieldEnum }
+    #       end), default_value: { direction: 'DESC', field: 'created_at' }
+
+    #       resolve ->(o, a, c) do
+    #         Resolvers::ExampleConnectionResolver.call o.example_connections, order_by: args.order_by
+    #       end
+    #     end
+    #   }
+    #   new = %{
+    #     field :example_connection, -> { ExampleConnectionType }, null: true, connection: true do
+    #       argument :order_by, (GraphQL::InputObjectType.define do
+    #         name 'ExampleConnectionOrder'
+    #         argument :direction, -> { ExampleConnectionDirectionEnum }, null: false
+    #         argument :field, -> { ExampleConnectionFieldEnum }, null: false
+    #       end), default_value: { direction: 'DESC', field: 'created_at' }
+
+    #       resolve ->(o, a, c) do
+    #         Resolvers::ExampleConnectionResolver.call o.example_connections, order_by: args.order_by
+    #       end
+    #     end
+    #   }
+
+    #   assert_equal upgrade(old), new
+    # end
+  end
+
+  describe 'multi-line field with property/method' do
+    it 'upgrades without breaking syntax' do
+      old = %{
+        field :is_example_field, types.Boolean,
+          property: :example_field?
+      }
+      new = %{
+        field :is_example_field, Boolean, null: true
+          method: :example_field?
+      }
+
+      assert_equal upgrade(old), new
+    end
+  end
+
+  describe 'multi-line connection with property/method' do
+    it 'upgrades without breaking syntax' do
+      old = %{
+        connection :example_connection, -> { ExampleConnectionType },
+          property: :example_connections
+      }
+      new = %{
+        field :example_connection, -> { ExampleConnectionType }, null: true, connection: true
+          method: :example_connections
+      }
+
+      assert_equal upgrade(old), new
+    end
+  end
+
+  # describe 'field with description inside block' do
+  #   it 'moves the description as an argument' do
+  #     old = %{
+  #       field :_id, !types.ID, property: :id do
+  #         description 'The primary key from the database'
```

Comment:
@esjee can we remove this from the PR and make a new one after you found out what's wrong with the description type?